src/main/kotlin/com/autonomousapps/model/Source.kt

```diff
@@ -104,7 +104,7 @@ data class AndroidResSource(
 
         val id = mapEntry.second
         return when {
-          id.startsWith('?') -> AttrRef(
+          ATTR_REGEX.containsMatchIn(id) -> AttrRef(
```

Comment:
Sorry for the long delay in re-reviewing. Work has been very busy, and I wanted time to think about this change, which is more subtle than might be obvious.

**tl;dr** I think this would be safer it it were `mapEntry.first != "android:text" && id.startsWith('?')`. WDYT? I'll explain some of the rationale in a comment directly on your new unit test.